targets/TARGET_ARM_FM/TARGET_FVP_MPS2/us_ticker.c

```diff
@@ -61,12 +87,10 @@ void us_ticker_set_interrupt(timestamp_t timestamp)
         us_ticker_init();
```

Comment:
This code is redundant and should be removed.

Reply:
So you mean it is not necessary to check the flag, we just need to call `us_ticker_init()` directly?

Reply:
No, I was thinking about entire block. Upper layer handles initialization, so it will call init only once on first ticker usage.

Reply:
I can see HAL API claimed that as undefined behavior, I will remove the function of checking flags